server/src/dtos/person.dto.ts

```diff
@@ -63,6 +63,19 @@ export class MergePersonDto {
 export class PersonSearchDto {
   @ValidateBoolean({ optional: true })
   withHidden?: boolean;
+
+  @ApiProperty({ description: 'Page number for pagination', default: 1, required: false })
```

Comment:
Pretty sure you can just use jsdoc comment and also default should come automatically from types

Reply:
Yep that works